.travis.yml

```diff
@@ -26,6 +27,7 @@ script:
 - travis-cargo --only stable test -- --features=deadlock_detection
 - travis-cargo --only beta test -- --features=deadlock_detection
 - travis-cargo --only nightly test -- --all --no-run --target x86_64-fortanix-unknown-sgx
+- travis-cargo --only nightly build -- --all --target x86_64-unknown-redox
```

Comment:
Should this be `--no-run`?

Reply:
It can't even link. That's why I do `build` instead of `test --no-run`.